lib/gc/ExecutionEngine/OpenCLRuntime/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+find_package(OpenCL REQUIRED)
+
+if(NOT OpenCL_FOUND)
+    message(FATAL_ERROR "OpenCL not found.")
+endif()
```

Comment:
```suggestion
```
This is redundant when the REQUIRED option is used.

Reply:
Thanks. Removed now.